src/ssh/_breaking_change.py

```diff
@@ -0,0 +1,3 @@
+from azure.cli.core.breaking_change import register_required_flag_breaking_change, register_default_value_breaking_change, register_other_breaking_change
+
+register_other_breaking_change('ssh arc', 'On May 21st 2025, any ssh arc commands on versions <2.0.4 will no longer work. Please upgrade to az ssh version >=2.0.4', taget_version='2.0.4')
```

Comment:
Should we detail the circumstances in which the older versions will be broken? Also "ssh vm" and "ssh config" might also break if they are being used for ARC machines